todo_app/app.py

```diff
@@ -1,11 +1,41 @@
-from flask import Flask
+from flask import Flask, redirect, url_for, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_item, get_items, add_item, save_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todo = get_items()
+    print(todo)
+    todo = sorted(todo, key=lambda x: x['status'], reverse=True)
+    print(todo)
```

Comment:
These print statements are really useful for debugging, but both add noise to the logs after and could potentially contain sensitive data (think of all the personal data potentially held in someone's todos!) so I'd recommend getting rid of these once you've got the feature working

Reply:
Done, I thought I had removed all print statements but missed these 2.  Thanks